src/_gettsim/sozialversicherung/arbeitslosen/arbeitslosengeld.py

```diff
@@ -1,22 +1,30 @@
 """Unemployment benefits (Arbeitslosengeld)."""
 
+from __future__ import annotations
+
 from _gettsim.einkommensteuer.einkommensteuer import einkommensteuertarif
 from ttsim import PiecewisePolynomialParameters, piecewise_polynomial, policy_function
 
 
-@policy_function(vectorization_strategy="loop")
+@policy_function(end_date="1998-07-31", leaf_name="betrag_m")
+def betrag_m_not_implemented() -> float:
+    """Calculate individual unemployment benefit."""
+    raise NotImplementedError("Not implemented yet.")
+
+
+@policy_function(vectorization_strategy="loop", start_date="1998-08-01")
```

Comment:
```suggestion
@policy_function(start_date="1998-08-01", vectorization_strategy="loop")
```
important things first :wink:. Though I don't even see why this cannot be vectorized, can probably be deleted with the change suggested below.